Sources/SwiftKuery/Utils.swift

```diff
@@ -112,4 +112,13 @@ struct Utils {
         resultQuery += inputQuery
         return (resultQuery, nameToNumber, index - 1)
     }
+
+    /// Escape apostrophes in SQL strings by doubling them.
+    ///
+    /// - Parameter value: The SQL string fragment in which to escape
+    ///   apostrophes.
+    /// - Returns: A string with apostrophes escaped by doubling them.
+    static func escapeApos(_ value: String) -> String {
```

Comment:
I think this would be better without truncating the method name, eg, escapeApostrophes.